Dockerfile

```diff
@@ -32,8 +32,7 @@ RUN set -x \
  && /fossology/install/scripts/php-conf-fix.sh --overwrite \
  && /fossology/utils/fo-installdeps -e -y \
  && $_cleanup
-RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer
+RUN make composer_install
```

Comment:
Wel, so we should remove it from fo-installdeps to let it only in the Makefile (or create a separate script).

Why do we need that: because for packaging, we don't need to invoke fo-installdeps, because deps are handled by packages. So we just need to handle what is missing, which is composer. So either through the Makefile or a specific script. But that one needs to be invoked from all other places instead of duplicating.